domdiv/main.py

```diff
@@ -971,17 +972,36 @@ def filter_sort_cards(cards, options):
     if options.expansion_dividers:
 
         cardnamesByExpansion = {}
+        randomizersByExpansion = {}
         for c in cards:
             if cardSorter.isBaseExpansionCard(c):
                 continue
-            cardnamesByExpansion.setdefault(c.cardset, []).append(c.name.strip().replace(' ', '&nbsp;'))
+            if c.randomizer:
+                randomizersByExpansion[c.cardset] = randomizersByExpansion.setdefault(c.cardset, 0) + 1
+            else:
+                randomizersByExpansion[c.cardset] = randomizersByExpansion.setdefault(c.cardset, 0)
+
+            if c.cardset not in cardnamesByExpansion:
+                cardnamesByExpansion[c.cardset] = {}
+            if c.card_tag not in cardnamesByExpansion[c.cardset]:
+                # Save off information about the card to be used on the expansion divider
+                order = 0
+                if c.card_tag in cardSorter.baseOrder:
+                    # Use the base card ordering
+                    order = 100 + cardSorter.baseOrder.index(c.card_tag)
+                cardnamesByExpansion[c.cardset][c.card_tag] = {'name': c.name.strip().replace(' ', '&nbsp;'),
+                                                               'randomizer': c.randomizer,
+                                                               'count': 1,
+                                                               'sort': "%03d%s" % (order, c.name.strip(),)}
+            else:
+                cardnamesByExpansion[c.cardset][c.card_tag]['count'] += 1
 
         for set_tag, set_values in Card.sets.iteritems():
             exp = set_values["set_name"]
             if exp in cardnamesByExpansion:
                 exp_name = exp
 
-                count = len(cardnamesByExpansion[exp])
+                count = randomizersByExpansion[exp]
```

Comment:
I'm not totally sure making the count here the randomizer count is intuitive without explanation. Maybe the count should stay the same as before, but the randomizer count get added somewhere else (end of the card list?) with some explanatory label?

Reply:
First, this is only for the expansion dividers.  Regular card dividers naturally have card counts that match the number of cards for that divider.  

But what is the best for the expansion dividers?  Now I admit I have more of a slip cover view, which is what I prefer to use.  So what does "count" mean for an expansion divider?  Currently, the count depends upon which groupings are selected.  You get one count if you group specials, another if you don't, and another if you move Events and Landmarks at the global level, etc.  The count varies.  None of these can really be used for slip covers - since there really are no cards of those types to hold in the slip cover.

What the slip covers *could* be used for is storing the randomizers for the expansion.   So at a minimum, I would propose using the randomizer as the number of cards used to generate the depth for the slip covers.   And right now, for all the non-expansion dividers in draw.py, that is the card count.

In the end, due to the variance, I thought it would be better to use the number of randomizers as the card count, and then just differentiate the cards in the card list by randomizer (Kingdom cards) vs non-supply cards.  So the count icon to quickly see if you have the right number of randomizers, and the list to see if you have all the right cards dividers.

Reply:
I see, ok. I don't have a problem with putting the number of randomizers, I just feel we're going to get bug reports saying "the number is wrong" if we don't explain it on the divider somehow. On the other hand, it's an optional (but nice) feature anyway, so maybe it's fine without explanation. Certainly not worth holding up the code over.